dvmconsole/MainWindow.xaml.cs

```diff
@@ -1613,6 +1614,18 @@ private void ToggleSnapCallHistory_Click(object sender, RoutedEventArgs e)
             settingsManager.SnapCallHistoryToWindow = !settingsManager.SnapCallHistoryToWindow;
         }
 
+        /// <summary>
+        /// 
+        /// </summary>
+        /// <param name="sender"></param>
+        /// <param name="e"></param>
+        private void ToggleStayOnTop_Click(object sender, RoutedEventArgs e)
+        {
+            this.Topmost = !this.Topmost;
```

Comment:
This event handler should include a check for if the window is loaded like ToggleSnapCallHistory_Click or ToggleLockWidgets_Click; otherwise during window load this will flip the settings flag incorrectly leading to a weird condition.